FBSnapshotTestCase/FBSnapshotTestCase.h

```diff
@@ -8,6 +8,8 @@
  *
  */
 
+#import <FBSnapshotTestCase/FBSnapshotTestCaseArchitecture.h>
```

Comment:
wdyt of platform? I suspect users will have sets of reference images for architecture, device, os version, maybe even locale.